packages/core/bootstrap/src/lib/cache/redis/redis.ts

```diff
@@ -105,25 +104,20 @@ export class RedisCache {
     )
   }
 
-  // TODO: We should have seperate services for response entries, and coalescing support
-  async setFlightMarker(key: string, maxAge: number) {
-    return this.contextualTimeout(this.client.set(key, true, { PX: maxAge }), 'setFlightMarker', {
-      key,
-      maxAge,
-    })
+  async setFlightMarker(): Promise<void> {
+    // NOTE: node-redis v4 supports auto-pipelining
+    // See https://github.com/redis/node-redis/issues/492
```

Comment:
Auto-pipelining != request coalescing (in our context at least), or am I missing anything?

I'm afraid removing this will break prod feeds if this was merged (DM me for more details).

Can we just disable request coalescing by default instead?